ipc.c

```diff
@@ -29,6 +29,19 @@ static long handle_reload(VarlinkService *service, VarlinkCall *call,
 	return 0;
 }
 
+static long select_profile(VarlinkService *service, VarlinkCall *call,
+		VarlinkObject *parameters, uint64_t flags, void *userdata) {
+	struct kanshi_state *state = userdata;
+	const char *profile;
+	if (varlink_object_get_string(parameters, "profile", &profile) < 0) {
+		return -1;
+	}
+	kanshi_set_profile(state, profile);
```

Comment:
Should we return an error if this fails?

Reply:
Hm, I guess the reload method doesn't return an error back to the IPC client either. Maybe something for a follow-up patch then.